made the default value of count to 8

PavamanSubramaniyam opened the pull request:
I have made the default value of count to 8 in the test case script ioping.py file also to be in sync with the yaml file parameter values

Also, created a README file inside ioping.py.data/ directory which was missing and moved the ioping Usage commented content lines in yaml file to this README file

Signed-off-by: Pavaman Subramaniyam <pavsubra@linux.vnet.ibm.com>

PavamanSubramaniyam commented:
ltcden11-lp13:/home/ioci/avocado-fvt-wrapper/tests/avocado-misc-tests/io/disk # avocado run --test-runner runner ioping.py -m ioping.py.data/ioping.yaml
Fetching asset from ioping.py:Ioping.test
JOB ID     : 8ae167d9c1372830aacee1b99c28520e9bdb8507
JOB LOG    : /home/ioci/avocado-fvt-wrapper/results/job-2022-05-23T01.17-8ae167d/job.log
 (1/2) ioping.py:Ioping.test;run-io_type-cache_io-d4ea: PASS (7.97 s)
 (2/2) ioping.py:Ioping.test;run-io_type-idirect_io-f5ad: PASS (8.00 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/ioci/avocado-fvt-wrapper/results/job-2022-05-23T01.17-8ae167d/results.html
JOB TIME   : 24.18 s

Naresh-ibm commented on io/disk/ioping.py.data/README:
```diff
@@ -0,0 +1,37 @@
+size - request size
+wsize - working set size
+disk - path for the disk
+
```
@PavamanSubramaniyam can u add little elaborated info for parameter like
`size` size of what? and what kind of request.
it will also more helpfull if you can add some generic example values.

abdhaleegit reviewed (approved):
LGTM

PavamanSubramaniyam commented:
Incorporated the review comments and added examples of the ioping command usage with different parameters in the README file and also added some generic example values for size, wsize and disk parameters passed in the ioping.yaml file.
